src/main/python/community-detector/community_detector.py

```diff
@@ -10,11 +9,8 @@
 def build_matrix(id_to_buckets):
     """Builds a CSR matrix from a list of lists of buckets
```

Comment:
Minor, but should this be changed? Now `id_to_buckets` is a "list of tuples (element_id, buckets)" right?